packages/mobile/src/components/user-list/UserCard.tsx

```diff
@@ -0,0 +1,85 @@
+import { useCallback } from 'react'
+
+import { SquareSizes, type ID } from '@audius/common/models'
+import { cacheUsersSelectors } from '@audius/common/store'
+import { formatCount } from '@audius/common/utils'
+import type { GestureResponderEvent } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import {
+  Avatar,
+  Divider,
+  Flex,
+  Paper,
+  Text,
+  type PaperProps
+} from '@audius/harmony-native'
+import { useNavigation } from 'app/hooks/useNavigation'
+
+import { useProfilePicture } from '../image/UserImage'
+import { UserLink } from '../user-link'
+
+const { getUser } = cacheUsersSelectors
+
+const messages = {
+  followers: (count: number) => (count === 1 ? 'Follower' : 'Followers')
```

Comment:
we have `pluralize('Follower', count)` but this is fine too

Reply:
oh great to know, will update